roles/config-python/defaults/main.yml

```diff
@@ -0,0 +1,3 @@
+---
+python_major_version: '3'
```

Comment:
this is unnecessary.
try:
```
python_version= {{ python_version }}
echo ${python_version:0:1}
```

Reply:
done. `major_version=`echo {{ python_version }} | cut -d '.' -f1``